### PR TITLE
bytesize talks nf-test and funcscan

### PR DESCRIPTION
**Pull request by FranBonath**
adding dates for the bytesize talks nf-test and funcscan.

<a href="https://gitpod.io/#https://github.com/nf-core/nf-co.re/pull/1474"></a>

### CODE REVIEW

**Review by mashehu — APPROVED** [review]

File: markdown/events/2023/bytesize_funcscan.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+Join us for our **weekly series** of short talks: **“nf-core/bytesize”**.
+
+Just **15 minutes** + questions, we will be focussing on topics about using and developing nf-core pipelines.
+These will be recorded and made available at <https://nf-co.re>
+It is our hope that these talks / videos will build an archive of training material that can complement our documentation. Got an idea for a talk? Let us know on the [`#bytesize`](https://nfcore.slack.com/channels/bytesize) Slack channel!
+
+## Bytesize: nf-core/funcscan
+
+This week, Jasmin Fragenberg ([@jasmezz](https://github.com/jasmezz)) is going to introduce nf-core/funcscan. nf-core/funcscan is a bioinformatics best-practice analysis pipeline for the screening of functional components of nucleotide sequences such as assembled contigs. This includes mining for antimicrobial peptides, antibiotic resistance genes and biosynthetic gene clusters.
```
Comment (jasmezz) [inline markdown/events/2023/bytesize_funcscan.md:23]:
```suggestion
This week, Jasmin Frangenberg ([@jasmezz](https://github.com/jasmezz)) is going to introduce nf-core/funcscan. nf-core/funcscan is a bioinformatics best-practice analysis pipeline for the screening of functional components of nucleotide sequences such as assembled contigs. This includes mining for antimicrobial peptides, antibiotic resistance genes and biosynthetic gene clusters.
```

File: markdown/events/2023/bytesize_funcscan.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
+---
+title: 'Bytesize: nf-core/funcscan'
+subtitle: Jasmin Fragenberg - Dept. Palaeobiotechnology, Leibniz Institute for Natural Product Research and Infection Biology Hans Knöll Institute
```
Comment (jasmezz) [inline markdown/events/2023/bytesize_funcscan.md:3]:
```suggestion
subtitle: Jasmin Frangenberg - Dept. Palaeobiotechnology, Leibniz Institute for Natural Product Research and Infection Biology Hans Knöll Institute
```

**Review by jasmezz — CHANGES_REQUESTED** [review]

**Comment by FranBonath** [comment]
@jasmezz sorry about the typo in the name (and then the copy paste error)!

**Review by jasmezz — APPROVED** [review]

**Review by Emiller88 — APPROVED** [review]
Looks good for mine!

**Pull request merged**